tests/foreman/virtwho/utils.py

```diff
@@ -303,3 +307,48 @@ def deploy_configure_by_script(script_content, debug=False):
         )
     if debug:
         return deploy_validation()
+
+
+def restart_virt_who_service():
+    """
+    Do the following:
+    1. clean rhsm.log message, make sure there is no old message exist.
+    2. restart virt-who service via systemctl command
+    """
+    runcmd("rm -f /var/log/rhsm/rhsm.log")
+    runcmd("systemctl restart virt-who; sleep 5")
+
+
+def update_configure_option(option, value, config_id):
+    """
+
+    :param option:
+    :param value:
+    :param config_id:
+    :return:
+    """
+    filename = get_configure_file(config_id)
+    cmd = 'sed -i "s|^{0}.*|{1}={2}|g" {3}'.format(
+        option, option, value, filename)
+    ret, output = runcmd(cmd)
+    if ret != 0:
+        raise VirtWhoError(
+            "Failed to set option {0} value to {1}".format(option, value))
+
+
+def delete_configure_option(option, config_id):
+    filename = get_configure_file(config_id)
+    cmd = 'sed -i "/^{0}/d" {1}; sed -i "/^#{2}/d" {3}'.format(
+        option, filename, option, filename)
```

Comment:
as above, 
1. using config_file to instead config_id
2. remove the duplicated format args